README.md

```diff
@@ -40,7 +40,6 @@ git clone https://github.com/dimagi/commcare-core.git
 - Click "OK" to use the Gradle wrapper
 - Wait while Android Studio spins its wheels
 - Download any build dependencies that the SDK Manager tells you you need.
-- Disable _Instant Run_ found in Settings > Build, Execution, Deployment > Instant Run. (It does not play well with multidexing, which we have enabled, or with some of the processes we have set up for Google Services)
```

Comment:
That menu doesn't seem to exist anymore, so I took the liberty to remove the whole line. I can just remove the part about Multidex if you prefer.